docs/bot_application_guide.rst

```diff
@@ -67,9 +67,9 @@ Enabling Privileged Intents
     .. image:: /.resources/bot-guide/discord_bot_tab.png
         :alt: The bot tab in the application page.
 
-5. Scroll down to the "Privileged Gateway Intents" section, enable both privileged intents and save your changes.
+5. Scroll down to the "Privileged Gateway Intents" section, enable all three privileged intents and save your changes. (Note: The "message content" intent must be enabled on or before August 31, 2022, or your bot will cease functioning once Discord begins to enforce it.)
```

Comment:
IMO this note is unnecessary - users should just always enable all three privileged intents and having this note could potentially make someone overthink it and not enable it.
```suggestion
5. Scroll down to the "Privileged Gateway Intents" section, enable all three privileged intents and save your changes.
```

Reply:
kk, I wasn't sure if someone might get confused about the screenshot but that's a good point, I removed it